examples/powercli/datastore-usage-email/README.md

```diff
@@ -16,7 +16,14 @@ git checkout master
 
 Step 2 - Update `stack.yml` and `vc-datastore-config.json` with your environment information
 
-<B>Note:</B> leave SMTP_USERNAME and SMTP_PASSWORD blank if you do not want to use authenticated SMTP
+**Note:** 
```

Comment:
Can you make these notes please in the form of

```
> **Note:** ...
```

Reply:
Meaning include the front > which makes it render like this: 

![image](https://user-images.githubusercontent.com/29929717/79612690-c2e4b280-80c2-11ea-8c13-81f9a9648db9.png)

Reply:
Fixed